hostname sb093.cluster
ipconfig 1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000 link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00 inet 127.0.0.1/8 scope host lo valid_lft forever preferred_lft forever inet6 ::1/128 scope host valid_lft forever preferred_lft forever 2: enp4s0f0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc mq state UP group default qlen 1000 link/ether a0:8c:f8:b9:a3:2e brd ff:ff:ff:ff:ff:ff inet 10.3.81.173/22 brd 10.3.83.255 scope global enp4s0f0 valid_lft forever preferred_lft forever inet6 fe80::a28c:f8ff:feb9:a32e/64 scope link valid_lft forever preferred_lft forever 3: enp4s0f1: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:b9:a3:2f brd ff:ff:ff:ff:ff:ff 4: enp4s0f2: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:b9:a3:30 brd ff:ff:ff:ff:ff:ff 5: enp4s0f3: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:b9:a3:31 brd ff:ff:ff:ff:ff:ff 6: ib0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 4092 qdisc pfifo_fast state UP group default qlen 256 link/infiniband 80:00:00:29:fe:80:00:00:00:00:00:00:38:bc:01:03:00:4c:f2:f0 brd 00:ff:ff:ff:ff:12:40:1b:ff:ff:00:00:00:00:00:00:ff:ff:ff:ff inet 10.3.89.173/22 brd 10.3.91.255 scope global ib0 valid_lft forever preferred_lft forever inet 10.3.93.173/22 brd 10.3.95.255 scope global ib0:1 valid_lft forever preferred_lft forever inet6 fe80::3abc:103:4c:f2f0/64 scope link valid_lft forever preferred_lft forever 7: ib1: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 4092 qdisc pfifo_fast state DOWN group default qlen 256 link/infiniband 80:00:00:29:fe:80:00:00:00:00:00:00:38:bc:01:03:00:4c:f2:f1 brd 00:ff:ff:ff:ff:12:40:1b:ff:ff:00:00:00:00:00:00:ff:ff:ff:ff
2024-01-19 04:57:41,080 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.3.89.173:34011'
2024-01-19 04:57:41,182 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.3.89.173:43952'
2024-01-19 04:57:43,704 - distributed.diskutils - INFO - Found stale lock file and directory '/scratch/dask-worker-space/worker-1jras8ip', purging
2024-01-19 04:57:43,707 - distributed.diskutils - INFO - Found stale lock file and directory '/scratch/dask-worker-space/worker-zy4md_oa', purging
2024-01-19 04:57:54,808 - distributed.worker - INFO -       Start worker at:    tcp://10.3.89.173:38424
2024-01-19 04:57:54,808 - distributed.worker - INFO -       Start worker at:    tcp://10.3.89.173:36447
2024-01-19 04:57:54,809 - distributed.worker - INFO -          Listening to:    tcp://10.3.89.173:38424
2024-01-19 04:57:54,809 - distributed.worker - INFO -          Listening to:    tcp://10.3.89.173:36447
2024-01-19 04:57:54,809 - distributed.worker - INFO -           Worker name:         SLURMCluster-473-1
2024-01-19 04:57:54,809 - distributed.worker - INFO -           Worker name:         SLURMCluster-473-0
2024-01-19 04:57:54,809 - distributed.worker - INFO -          dashboard at:          10.3.89.173:45941
2024-01-19 04:57:54,809 - distributed.worker - INFO -          dashboard at:          10.3.89.173:42743
2024-01-19 04:57:54,809 - distributed.worker - INFO - Waiting to connect to:     tcp://10.3.88.12:42763
2024-01-19 04:57:54,809 - distributed.worker - INFO - Waiting to connect to:     tcp://10.3.88.12:42763
2024-01-19 04:57:54,809 - distributed.worker - INFO - -------------------------------------------------
2024-01-19 04:57:54,809 - distributed.worker - INFO - -------------------------------------------------
2024-01-19 04:57:54,809 - distributed.worker - INFO -               Threads:                          2
2024-01-19 04:57:54,809 - distributed.worker - INFO -               Threads:                          2
2024-01-19 04:57:54,810 - distributed.worker - INFO -                Memory:                  10.24 GiB
2024-01-19 04:57:54,810 - distributed.worker - INFO -                Memory:                  10.24 GiB
2024-01-19 04:57:54,810 - distributed.worker - INFO -       Local Directory: /scratch/dask-worker-space/worker-_zp0cyl3
2024-01-19 04:57:54,810 - distributed.worker - INFO -       Local Directory: /scratch/dask-worker-space/worker-24l1e91s
2024-01-19 04:57:54,810 - distributed.worker - INFO - -------------------------------------------------
2024-01-19 04:57:54,810 - distributed.worker - INFO - -------------------------------------------------
2024-01-19 04:57:54,872 - distributed.worker - INFO -         Registered to:     tcp://10.3.88.12:42763
2024-01-19 04:57:54,873 - distributed.worker - INFO - -------------------------------------------------
2024-01-19 04:57:54,873 - distributed.core - INFO - Starting established connection to tcp://10.3.88.12:42763
2024-01-19 04:57:54,876 - distributed.worker - INFO -         Registered to:     tcp://10.3.88.12:42763
2024-01-19 04:57:54,876 - distributed.worker - INFO - -------------------------------------------------
2024-01-19 04:57:54,877 - distributed.core - INFO - Starting established connection to tcp://10.3.88.12:42763
2024-01-19 05:38:15,641 - distributed.core - INFO - Event loop was unresponsive in Worker for 20.79s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/__init__.py:55: UserWarning: Dependency not satisfied, torchani.ase will not be available
  warnings.warn("Dependency not satisfied, torchani.ase will not be available")
Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 7 conformers. 
Removed 3 conformers. 
Generated 35 conformers. 
Removed 27 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/4 [00:00<?, ?it/s][W BinaryOps.cpp:601] Warning: floor_divide is deprecated, and will be removed in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values.
To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor'). (function operator())
Optimising conformer:  25%|█████▊                 | 1/4 [00:02<00:08,  2.77s/it]Optimising conformer:  50%|███████████▌           | 2/4 [00:04<00:04,  2.07s/it]Optimising conformer:  75%|█████████████████▎     | 3/4 [00:06<00:01,  1.89s/it]Optimising conformer: 100%|███████████████████████| 4/4 [00:07<00:00,  1.77s/it]Optimising conformer: 100%|███████████████████████| 4/4 [00:07<00:00,  1.90s/it]
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/8 [00:00<?, ?it/s]Optimising conformer:  12%|██▉                    | 1/8 [00:03<00:24,  3.50s/it]Optimising conformer:  25%|█████▊                 | 2/8 [00:05<00:16,  2.67s/it]Optimising conformer:  38%|████████▋              | 3/8 [00:07<00:11,  2.39s/it]Optimising conformer:  50%|███████████▌           | 4/8 [00:09<00:09,  2.25s/it]Optimising conformer:  62%|██████████████▍        | 5/8 [00:11<00:06,  2.18s/it]Optimising conformer:  75%|█████████████████▎     | 6/8 [00:13<00:04,  2.11s/it]Optimising conformer:  88%|████████████████████▏  | 7/8 [00:15<00:02,  2.08s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:17<00:00,  2.05s/it]Optimising conformer: 100%|███████████████████████| 8/8 [00:17<00:00,  2.21s/it]
2024-01-19 06:15:31,416 - distributed.utils_perf - INFO - full garbage collection released 10.25 MiB from 466037 reference cycles (threshold: 9.54 MiB)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME Completed the molecule generation in 48.9s.
TIME Completed the molecule generation in 69.4s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 15 conformers. 
Generated 25 conformers. 
Removed 12 conformers. 
Removed 7 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/3 [00:00<?, ?it/s]Optimising conformer:  33%|███████▋               | 1/3 [00:02<00:05,  2.84s/it]Optimising conformer:  67%|███████████████▎       | 2/3 [00:03<00:01,  1.84s/it]Optimising conformer: 100%|███████████████████████| 3/3 [00:06<00:00,  2.03s/it]Optimising conformer: 100%|███████████████████████| 3/3 [00:06<00:00,  2.08s/it]
TIME Completed the molecule generation in 62.1s.
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                              | 0/18 [00:00<?, ?it/s]Optimising conformer:   6%|█▏                    | 1/18 [00:03<01:06,  3.91s/it]Optimising conformer:  11%|██▍                   | 2/18 [00:45<06:56, 26.05s/it]Optimising conformer:  17%|███▋                  | 3/18 [00:48<03:52, 15.48s/it]Optimising conformer:  22%|████▉                 | 4/18 [00:50<02:24, 10.33s/it]Optimising conformer:  28%|██████                | 5/18 [00:53<01:40,  7.76s/it]Optimising conformer:  33%|███████▎              | 6/18 [00:57<01:15,  6.26s/it]Optimising conformer:  39%|████████▌             | 7/18 [01:00<00:55,  5.09s/it]Optimising conformer:  44%|█████████▊            | 8/18 [01:04<00:47,  4.76s/it]Optimising conformer:  50%|███████████           | 9/18 [01:07<00:38,  4.30s/it]Optimising conformer:  56%|███████████▋         | 10/18 [01:10<00:31,  3.96s/it]Optimising conformer:  61%|████████████▊        | 11/18 [01:12<00:23,  3.41s/it]Optimising conformer:  67%|██████████████       | 12/18 [01:16<00:20,  3.46s/it]Optimising conformer:  72%|███████████████▏     | 13/18 [01:18<00:16,  3.20s/it]Optimising conformer:  78%|████████████████▎    | 14/18 [01:21<00:12,  3.07s/it]Optimising conformer:  83%|█████████████████▌   | 15/18 [01:24<00:08,  2.98s/it]Optimising conformer:  89%|██████████████████▋  | 16/18 [01:27<00:05,  2.94s/it]Optimising conformer:  94%|███████████████████▊ | 17/18 [01:30<00:02,  2.90s/it]Optimising conformer: 100%|█████████████████████| 18/18 [02:18<00:00, 16.70s/it]Optimising conformer: 100%|█████████████████████| 18/18 [02:18<00:00,  7.72s/it]
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME Completed the molecule generation in 256.5s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 21 conformers. 
Removed 10 conformers. 
Generated 32 conformers. 
Removed 19 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                              | 0/11 [00:00<?, ?it/s]Optimising conformer:   9%|██                    | 1/11 [00:03<00:30,  3.01s/it]Optimising conformer:  18%|████                  | 2/11 [00:08<00:39,  4.35s/it]using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'

Optimising conformer:   0%|                              | 0/13 [00:00<?, ?it/s][AOptimising conformer:  27%|██████                | 3/11 [00:10<00:25,  3.18s/it]
Optimising conformer:   8%|█▋                    | 1/13 [00:02<00:33,  2.76s/it][AOptimising conformer:  36%|████████              | 4/11 [00:12<00:19,  2.78s/it]
Optimising conformer:  15%|███▍                  | 2/13 [00:05<00:30,  2.78s/it][A
Optimising conformer:  23%|█████                 | 3/13 [00:06<00:21,  2.13s/it][A
Optimising conformer:  31%|██████▊               | 4/13 [00:08<00:16,  1.80s/it][AOptimising conformer:  45%|██████████            | 5/11 [00:17<00:21,  3.61s/it]
Optimising conformer:  38%|████████▍             | 5/13 [00:09<00:13,  1.65s/it][AOptimising conformer:  55%|████████████          | 6/11 [00:19<00:15,  3.11s/it]Optimising conformer:  64%|██████████████        | 7/11 [00:21<00:10,  2.60s/it]
Optimising conformer:  46%|██████████▏           | 6/13 [00:14<00:19,  2.84s/it][A
Optimising conformer:  54%|███████████▊          | 7/13 [00:16<00:14,  2.36s/it][A
Optimising conformer:  62%|█████████████▌        | 8/13 [00:17<00:10,  2.01s/it][AOptimising conformer:  73%|████████████████      | 8/11 [00:27<00:11,  3.96s/it]
Optimising conformer:  69%|███████████████▏      | 9/13 [00:20<00:09,  2.34s/it][A
Optimising conformer:  77%|████████████████▏    | 10/13 [00:21<00:06,  2.04s/it][AOptimising conformer:  82%|██████████████████    | 9/11 [00:31<00:07,  3.84s/it]
Optimising conformer:  85%|█████████████████▊   | 11/13 [00:23<00:03,  1.85s/it][AOptimising conformer:  91%|███████████████████  | 10/11 [00:33<00:03,  3.27s/it]
Optimising conformer:  92%|███████████████████▍ | 12/13 [00:24<00:01,  1.73s/it][A
Optimising conformer: 100%|█████████████████████| 13/13 [00:26<00:00,  1.62s/it][AOptimising conformer: 100%|█████████████████████| 13/13 [00:26<00:00,  2.00s/it]
Optimising conformer: 100%|█████████████████████| 11/11 [00:39<00:00,  4.02s/it]Optimising conformer: 100%|█████████████████████| 11/11 [00:39<00:00,  3.56s/it]
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
TIME Completed the molecule generation in 99.4s.
TIME Completed the molecule generation in 106.8s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 45 conformers. 
Removed 36 conformers. 
Generated 48 conformers. 
Removed 39 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/9 [00:00<?, ?it/s]Optimising conformer:  11%|██▌                    | 1/9 [00:04<00:33,  4.21s/it]Optimising conformer:  22%|█████                  | 2/9 [00:06<00:22,  3.20s/it]Optimising conformer:  33%|███████▋               | 3/9 [00:10<00:21,  3.50s/it]Optimising conformer:  44%|██████████▏            | 4/9 [00:12<00:15,  3.06s/it]using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'

Optimising conformer:   0%|                               | 0/9 [00:00<?, ?it/s][A
Optimising conformer:  11%|██▌                    | 1/9 [00:03<00:25,  3.20s/it][A
Optimising conformer:  22%|█████                  | 2/9 [00:05<00:17,  2.45s/it][AOptimising conformer:  56%|████████████▊          | 5/9 [00:20<00:18,  4.73s/it]
Optimising conformer:  33%|███████▋               | 3/9 [00:06<00:12,  2.04s/it][A
Optimising conformer:  44%|██████████▏            | 4/9 [00:08<00:09,  1.99s/it][AOptimising conformer:  67%|███████████████▎       | 6/9 [00:23<00:11,  3.95s/it]
Optimising conformer:  56%|████████████▊          | 5/9 [00:10<00:07,  1.95s/it][A
Optimising conformer:  67%|███████████████▎       | 6/9 [00:12<00:05,  1.82s/it][AOptimising conformer:  78%|█████████████████▉     | 7/9 [00:26<00:07,  3.94s/it]
Optimising conformer:  78%|█████████████████▉     | 7/9 [00:13<00:03,  1.74s/it][A
Optimising conformer:  89%|████████████████████▍  | 8/9 [00:15<00:01,  1.80s/it][A
Optimising conformer: 100%|███████████████████████| 9/9 [00:17<00:00,  1.84s/it][AOptimising conformer: 100%|███████████████████████| 9/9 [00:17<00:00,  1.94s/it]
Optimising conformer:  89%|████████████████████▍  | 8/9 [00:35<00:05,  5.51s/it]Optimising conformer: 100%|███████████████████████| 9/9 [00:39<00:00,  5.04s/it]Optimising conformer: 100%|███████████████████████| 9/9 [00:39<00:00,  4.43s/it]
2024-01-19 09:24:31,996 - distributed.worker - ERROR - Failed to communicate with scheduler during heartbeat.
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/tornado/iostream.py", line 861, in _read_to_buffer
    bytes_read = self.read_from_fd(buf)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/tornado/iostream.py", line 1116, in read_from_fd
    return self.socket.recv_into(buf, len(buf))
TimeoutError: [Errno 110] Connection timed out

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 1237, in heartbeat
    response = await retry_operation(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/utils_comm.py", line 434, in retry_operation
    return await retry(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/utils_comm.py", line 413, in retry
    return await coro()
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1227, in send_recv_from_rpc
    return await send_recv(comm=comm, op=key, **kwargs)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 986, in send_recv
    response = await comm.read(deserializers=deserializers)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 241, in read
    convert_stream_closed_error(self, e)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc.__class__.__name__}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) ConnectionPool.heartbeat_worker local=tcp://10.3.89.173:45804 remote=tcp://10.3.88.12:42763>: TimeoutError: [Errno 110] Connection timed out
2024-01-19 09:24:31,996 - distributed.worker - ERROR - Failed to communicate with scheduler during heartbeat.
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/tornado/iostream.py", line 861, in _read_to_buffer
    bytes_read = self.read_from_fd(buf)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/tornado/iostream.py", line 1116, in read_from_fd
    return self.socket.recv_into(buf, len(buf))
TimeoutError: [Errno 110] Connection timed out

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 1237, in heartbeat
    response = await retry_operation(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/utils_comm.py", line 434, in retry_operation
    return await retry(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/utils_comm.py", line 413, in retry
    return await coro()
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1227, in send_recv_from_rpc
    return await send_recv(comm=comm, op=key, **kwargs)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 986, in send_recv
    response = await comm.read(deserializers=deserializers)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 241, in read
    convert_stream_closed_error(self, e)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc.__class__.__name__}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) ConnectionPool.heartbeat_worker local=tcp://10.3.89.173:45806 remote=tcp://10.3.88.12:42763>: TimeoutError: [Errno 110] Connection timed out
2024-01-19 09:24:46,460 - distributed.core - INFO - Connection to tcp://10.3.88.12:42763 has been closed.
2024-01-19 09:24:46,460 - distributed.core - INFO - Connection to tcp://10.3.88.12:42763 has been closed.
2024-01-19 09:24:46,461 - distributed.worker - INFO - Stopping worker at tcp://10.3.89.173:38424. Reason: worker-handle-scheduler-connection-broken
2024-01-19 09:24:46,461 - distributed.worker - INFO - Stopping worker at tcp://10.3.89.173:36447. Reason: worker-handle-scheduler-connection-broken
2024-01-19 09:24:46,468 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.3.89.173:43952'. Reason: worker-handle-scheduler-connection-broken
2024-01-19 09:24:46,468 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.3.89.173:34011'. Reason: worker-handle-scheduler-connection-broken
2024-01-19 09:24:46,481 - distributed.worker - ERROR - Failed to communicate with scheduler during heartbeat.
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 291, in connect
    comm = await asyncio.wait_for(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 466, in wait_for
    await waiter
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 1237, in heartbeat
    response = await retry_operation(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/utils_comm.py", line 434, in retry_operation
    return await retry(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/utils_comm.py", line 413, in retry
    return await coro()
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1224, in send_recv_from_rpc
    comm = await self.pool.connect(self.addr)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1412, in _connect
    raise CommClosedError(reason)
distributed.comm.core.CommClosedError: ConnectionPool closing.
2024-01-19 09:24:46,502 - distributed.nanny - INFO - Worker closed
2024-01-19 09:24:46,491 - distributed.worker - ERROR - Failed to communicate with scheduler during heartbeat.
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 291, in connect
    comm = await asyncio.wait_for(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 466, in wait_for
    await waiter
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 1237, in heartbeat
    response = await retry_operation(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/utils_comm.py", line 434, in retry_operation
    return await retry(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/utils_comm.py", line 413, in retry
    return await coro()
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1224, in send_recv_from_rpc
    comm = await self.pool.connect(self.addr)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1412, in _connect
    raise CommClosedError(reason)
distributed.comm.core.CommClosedError: ConnectionPool closing.
2024-01-19 09:24:46,513 - distributed.nanny - INFO - Worker closed
2024-01-19 09:24:48,513 - distributed.nanny - ERROR - Worker process died unexpectedly
2024-01-19 09:24:48,520 - distributed.nanny - ERROR - Worker process died unexpectedly
TIME Completed the molecule generation in 124.3s.
TIME Completed the molecule generation in 136.4s.
2024-01-19 09:24:49,588 - distributed.nanny - INFO - Closing Nanny at 'tcp://10.3.89.173:43952'. Reason: nanny-close-gracefully
2024-01-19 09:24:52,073 - distributed.nanny - INFO - Closing Nanny at 'tcp://10.3.89.173:34011'. Reason: nanny-close-gracefully
2024-01-19 09:24:52,075 - distributed.dask_worker - INFO - End worker
